edit_call_section.m

```diff
@@ -260,24 +260,63 @@ function reload_bp_timeseries(hh,mic_config)
 if any(isnan(se_idx))
     call_short = NaN;
 else
-    call_short = gui_call_op.call_align(se_idx(1):se_idx(2),gui_call_op.curr_ch)';  % extract new short call segment
+    call_short = gui_call_op.call_align(se_idx(1):se_idx(2),gui_call_op.curr_ch);  % extract new short call segment
 end
-w = tukeywin(length(call_short),tukeywin_prop);
-call_short_taper = call_short.*w';  % taper call for spectrum estimation
+w = tukeywin(size(call_short,1),tukeywin_prop);
+call_short_taper = call_short.*repmat(w,1,size(call_short,2));  % taper call for spectrum estimation
+
 call_fft = fft(call_short_taper);
-call_freq_vec = linspace(0,data.mic_data.fs/2,round((length(call_fft)+1)/2));
-call_fft_len = length(call_freq_vec);
-% call_psd = 2*abs(call_fft(1:call_fft_len)).^2/(length(call_fft)*data.mic_data.fs);
-call_psd = 2*abs(call_fft(1:call_fft_len)).^2;

```

Comment:
In this version `call_psd` should really be `call_esd`, where psd is power spectral density and esd is energy spectral density.